pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -726,9 +730,26 @@ func configureVifV6Addresses(b *MasqueradeBindMechanism, err error) error {
 }
 
 func (b *MasqueradeBindMechanism) startDynamicIPServers(vmi *v1.VirtualMachineInstance) error {
+	if err := b.startRADaemon(); err != nil {
+		log.Log.Criticalf("could not start the RA daemon: %v", err)
+		return err
+	}
 	return Handler.StartDHCP(b.vif, b.vif.Gateway, b.bridgeInterfaceName, b.iface.DHCPOptions, false)
 }
 
+func (b *MasqueradeBindMechanism) startRADaemon() error {
+	targetPID := "self"
+	if err := Handler.CreateRADaemon(
+		GetNDPConnectionUnixSocketPath(targetPID, b.bridgeInterfaceName),
+		b.bridgeInterfaceName,
+		api.DefaultVMIpv6CIDR,
+		5); err != nil {
```

Comment:
Can you please explain why with the process the retry mechanism was needed and with the go-routine it isn't?

Reply:
From your explanation I understand that the chances to need a retry were significantly reduced. But on a "slow" virt-handler we may still need it. Don't we?

Reply:
I would add a timeout anyway.

```
ctx, cancel := context.WithTimeout(context.Background(), time.Minute)
defer cancel()

conn, err := d.DialContext(ctx, ...)
```

Reply:
Done.